plugin-server/src/utils/status.ts

```diff
@@ -61,7 +61,7 @@ export class Status implements StatusBlueprint {
 
     buildMethod(type: keyof StatusBlueprint): StatusMethod {
         return (icon: string, message: string, extra: object) => {
-            const logMessage = `(${this.mode}) ${icon} ${message}`
+            const logMessage = `${icon} ${message}`
```

Comment:
Is this because mode is currently `undefined`? 

I'd love to fix whatever broke this as right now I can't see what worker thread is doing what

Reply:
Yes 😳  I'll fix 😄